src/v2/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -234,22 +252,47 @@ export class OfferRoute extends Component<OfferProps, OfferState> {
               style={isCommittingMutation ? { pointerEvents: "none" } : {}}
               id="offer-page-left-column"
             >
-              <Flex flexDirection="column">
-                <OfferInput
-                  id="OfferForm_offerValue"
-                  showError={
-                    this.state.formIsDirty && this.state.offerValue <= 0
-                  }
-                  onChange={offerValue => this.setState({ offerValue })}
-                  onFocus={this.onOfferInputFocus.bind(this)}
-                />
-              </Flex>
-              {Boolean(offerItem?.price) && (
-                <Text my={1} variant="xs" color="black60">
-                  List price:{" "}
-                  {appendCurrencySymbol(offerItem?.price, order.currencyCode)}
-                </Text>
+              {(!newOfferSubmissionEnabled || isInquiryCheckout) && (
```

Comment:
the name here `isInquiryCheckout` is a bit dubious to me, I was reading thinking that's is the offer inquiry but then I re-checked and the variable meaning is if the artwork has no price/priceRange, can you please change it to a proper explanatory name?
I don't have a strong suggestion but would stick to something related to the meaning, `hasNoPrice`, etc

Reply:
Yeah, this is a tricky one because, sure `hasNoPrice`  makes sense, but context-wise, the only reason that the user would be seeing this component and a price wouldn’t be present is if the offer was started through conversations and hence: `isInquiryCheckout`.

Reply:
Ah okay, so when we're in the `Offer` submission flow given no price nor listPrice we consider it `InquiryCheckout`, right?
Can you clarify the difference between Inquiry and Offer, please? 🙏

Reply:
Of course!
We consider an "Inquiry" when the collector clicks on a "Contact Gallery" button on the artwork page. If the user then clicks on "Make Offer" from within that inquiry/conversation, then we consider that offer to be "inquiry checkout", which you'll also see it referred as "MOI" => "Make Offer on Inquiry".

A regular offer, or a non-MOI, is one where the conversation tab wasn't involved to start the offer; the user clicked on "Make Offer" directly on the artwork page.